Formula/c/csvkit.rb

```diff
@@ -122,6 +122,7 @@ class Csvkit < Formula
 
   def install
     virtualenv_install_with_resources
+    man1.install_symlink libexec.glob("share/man/man1/*.1")
```

Comment:
```suggestion
    virtualenv_install_with_resources(link_manpages: true)
```

Also need to update commit message to start with formula name, e.g. `csvkit: link newly included manpages`